xio/src/test/timeout_test.c

```diff
@@ -25,7 +25,7 @@ static globus_bool_t                    globus_l_closed = GLOBUS_FALSE;
 
 static globus_bool_t                    globus_l_timeout = GLOBUS_TRUE;
 
-#define USEC_THRESHHOLD  500000
+#define USEC_THRESHHOLD  50000
```

Comment:
Alternatively, we can extend the travis timeout with [travis_wait](https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received)

Reply:
this is 500ms versus 50ms, right?  Why are we so sensitive to this?

@matyasselmeci - how long does this take to run without this change?